### ISSUE TITLE
dependabot commonmarker

### ISSUE THREAD

**Issue by ilessing** [UCSBCarpentry/ucsbcarpentry.github.io#11]
![Screen Shot 2022-03-04 at 12 29 51 PM](https://user-images.githubusercontent.com/275899/156837135-2f88b04d-cded-4222-8ff6-692ee93a4326.png)

**Comment by ilessing** [comment]
Examined the `Gemfile` which led me to https://pages.github.com/versions
from there I deduced that we can use a more recent version of commonmark gem

visited: https://www.rubydoc.info/gems/jekyll-commonmark-ghpages/0.2.0

which suggests adding the following to the `Gemfile`:
```
group :jekyll_plugins do
  gem 'jekyll-commonmark-ghpages'
end
```

**Comment by ilessing** [comment]
crap.   now the "past workshops" page is broken.

**Comment by ilessing** [comment]
I reverted the previous commit: e126452

**Comment by ilessing** [comment]
@kristi-sara & @apandas 
I had a go at fixing this and ran into some issues.  I'll try again later but feel free to pick at this one if you have time.  HA.. like time weren't in scarce supply.

**Comment by ilessing** [comment]
never mind.   I think I've sorted it now.
see:  https://github.com/UCSBCarpentry/ucsbcarpentry.github.io/security/dependabot/2

what I did:
- deleted the file: `Gemfile.lock`
- rebuilt it with `bundle install`

That updated a bunch of dependencies listed in the `Gemfile.lock`

- committed the changes: 02caace3ad1f
- and pushed back to Github

I think that solved it.
Re-open this issue if you spot any further problems.
